src/umpire/strategy/mixins/AlignedAllocation.hpp

```diff
@@ -0,0 +1,58 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#ifndef UMPIRE_aligned_allocation_HPP
+#define UMPIRE_aligned_allocation_HPP
+
+#include "umpire/strategy/AllocationStrategy.hpp"
+
+#include <unordered_map>
+
+namespace umpire {
+namespace strategy {
+namespace mixins {
+
+class AlignedAllocation {
+public:
+    AlignedAllocation() = delete;
+    AlignedAllocation(std::size_t alignment, strategy::AllocationStrategy* strategy);
+
+    //////////////////////////////////////////////////////////////////////////
+    /// \brief Round up the size to be an integral multple of configured
```

Comment:
You should use `/*!` for Doxygen comments.

Reply:
Done.

As discussed, I changed to:
//!
//! ... text ...
//!

Reply:
Done.  Please mark resolved if you agree.